### ISSUE TITLE
Add the option to toggle NV in Phase Immersion suit.

### ISSUE THREAD

**Issue by Jan-Blasiak** [CleverRaven/Cataclysm-DDA#74345]
### Is your feature request related to a problem? Please describe.

Sometimes you just want to resist the portal storm effects or just stay warm/cool in inhospitable weather. Currently you can only do that when you ALSO have NV overlay, which is very annoying when its bright outside or you're in a brightly lit interior.

### Solution you would like.

A toggle like with HUB-01 recon headgear.

### Describe alternatives you have considered.

A quick menu to select desired state?

### Additional context

QOL change.

**Comment by Jan-Blasiak** [comment]
> You can disable the green color in settings > graphics > night vision color overlay

Yes, but that's not an immersive, in-character way to do it, that's just workaround.

**Comment by anoobindisguise** [comment]
> Yes, but that's not an immersive, in-character way to do it, that's just workaround.

that doesn't seem like a very solid argument to add a big extra layer of complexity to the armor. since having active NV doesn't actually impede you in any way when using them in daylight. IRL, NV goggles basically become blinded while under sunlight, but in game there's no consequence. If there actually was a bloom-out mechanic then it would be justified maybe but rn it's gamified enough that you can just turn off green mode and ignore it imo

**Comment by John-Candlebury** [comment]
It would be neat if the suit's night vision toggled itself automatically based on ambient light if it ever becomes necessary to account for that.
